src/__tests__/index.js

```diff
@@ -353,8 +353,10 @@ test('reads text from fs if filePath is provided but not text', () => {
   format({ filePath });
   // format({filePath}).catch(() => {})
   // one hit to get the file and one for the eslintignore
-  expect(fsMock.readFileSync).toHaveBeenCalledTimes(2);
+  expect(fsMock.readFileSync).toHaveBeenCalledTimes(3);
```

Comment:
Apparently any call to CLIEngine(eslintOptions) where eslintOptions doesn't contain a cwd nor a ignore set to false, will lead to reading the eslintignore twice now.